src/http-client.js

```diff
@@ -64,3 +69,27 @@ function getSortedQS (params) {
     return acc.concat([qs.stringify(pair)]);
   }, []).join('&');
 }
+
+function getUserAgent () {
+  const app = 'app contentful.cf-graphql';
+  return [app, getOs(), getPlatform()].filter(s => typeof s === 'string').join('; ');
```

Comment:
This is probably just my OCD talking but I'd `trim()` the final result to get rid of that extra space. 😄